charts/posthog/values.yaml

```diff
@@ -552,9 +552,6 @@ clickhouse:
   # -- Service Type: LoadBalancer (allows external access) or NodePort (more secure, no extra cost)
   serviceType: NodePort
 
-  # -- If enabled, operator will prefer k8s nodes with tag `clickhouse:true`
-  useNodeSelector: false
```

Comment:
can we get `nodeSelector` documented in values it currently isn't (i.e. isn't in ALL_VALUES)